packages/auth-client/src/controllers/engine.ts

```diff
@@ -191,6 +194,41 @@ export class AuthEngine extends IAuthEngine {
     return pendingRequests;
   };
 
+  public formatMessage = (cacao: AuthEngineTypes.CacaoPayload, iss: string) => {
+    this.client.logger.debug("constructEip4361Message, cacao is:", cacao);
```

Comment:
Aligning naming in debug log:

```suggestion
    this.client.logger.debug("formatMessage, cacao is:", cacao);
```